src/specs/components/picture/demo/data.js

```diff
@@ -0,0 +1,27 @@
+// Simple content for demo
+module.exports = {
+  picture: {
+    img: {
+      src: 'https://inno-ecl.s3.amazonaws.com/media/examples/example-image.jpg',
+      alt: 'Image alternative text',
+    },
+    sources: [
+      {
+        src: 'https://inno-ecl.s3.amazonaws.com/media/examples/example-image5.jpg',
+        media: 'xl',
+      },
+      {
+        src: 'https://inno-ecl.s3.amazonaws.com/media/examples/example-image4.jpg',
+        media: 'l',
+      },
+      {
+        src: 'https://inno-ecl.s3.amazonaws.com/media/examples/example-image3.jpg',
+        media: 'm',
+      },
+      {
+        src: 'https://inno-ecl.s3.amazonaws.com/media/examples/example-image2.jpg',
+        media: 's',
```

Comment:
weren't you defining a source with a custom media query before..? I think we should have that in this data we use the test the picture template